pars:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  

  # Velocity Controllers ---------------------------------------
  left_rear_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: pars__jRL
    pid: {p: 1.0, i: 0.0, d: 0.0, i_clamp: 0.0}
  right_rear_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: pars__jRR
    pid: {p: 1.0, i: 0.0, d: 0.0, i_clamp: 0.0}
  left_front_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: pars__jFL
    pid: {p: 0.5, i: 0.0, d: 0.0, i_clamp: 0.0}
  right_front_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: pars__jFR
    pid: {p: 0.5, i: 0.0, d: 0.0, i_clamp: 0.0}


  # Position Controllers ---------------------------------------
  left_steering_hinge_position_controller:
    joint: pars__jFLsteer
    type: effort_controllers/JointPositionController
    pid: {p: 1.0, i: 0.0, d: 0.5}
  right_steering_hinge_position_controller:
    joint: pars__jFRsteer
    type: effort_controllers/JointPositionController
    pid: {p: 1.0, i: 0.0, d: 0.5}

